user_sync/connector/directory_okta.py

```diff
@@ -185,137 +179,103 @@ def iter_group_members(self, group):
         '''
         res_group = self.find_group(group)
         if (res_group != None):
-
-            members = self.groups_client.get_group_users(res_group.id)
-
+            try:
+                members = self.groups_client.get_group_users(res_group.id)
+            except Exception as e:
+                self.logger.warning("Unable to get_group_users")
+                raise user_sync.error.AssertionException(repr(e))
             for member in members:
                 yield (member.id)
-
         else:
             self.logger.warning("No group found for: %s", group)
 
-
     def iter_users(self, users_filter, extended_attributes):
         options = self.options
 
         ###TODO does nothing for now
-        user_attribute_names = ["firstName", "lastName", "countryCode", "id"]
-        user_attribute_names.extend(self.user_email_formatter.get_attribute_names())
-        user_attribute_names.extend(self.user_username_formatter.get_attribute_names())
-        user_attribute_names.extend(self.user_domain_formatter.get_attribute_names())
+        user_attribute_names = ["firstName", "lastName","login","email" "countryCode"]
```

Comment:
Looks like you are missing a comma after "email".  Doing that does not appear to be invalid Python, but will yield unexpected results.

```python
>>> user_attribute_names = ["firstName", "lastName","login","email" "countryCode"]
>>> print user_attribute_names
['firstName', 'lastName', 'login', 'emailcountryCode']
```